pom.xml

```diff
@@ -68,7 +68,7 @@
 
         <!-- versions -->
         <zookeeper-version>3.6.0</zookeeper-version>
-        <maven-bundle-plugin-version>4.1.0</maven-bundle-plugin-version>
+        <maven-bundle-plugin-version>5.1.1</maven-bundle-plugin-version>
```

Comment:
Yeah, I changed this because i was getting build errors trying to build curator locally.
This was the recommended fix I found, and it did fix it for me. I am also not a maven expert so if there is a different or better way to fix it I'm happy to change it.

Reply:
It should be fine to make the upgrade